src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -404,6 +403,7 @@
   </ItemGroup>
   <!-- Common (Windows and Unix) dependencies for MANAGED_SNI build -->
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(OSGroup)' != 'AnyOS'">
+    <Reference Include="System.Memory" />
```

Comment:
@safern, should this be under the ItemGroup that is just `Condition="'$(IsPartialFacadeAssembly)' != 'true'`? Why do we need to avoid adding the reference to System.Memory for AnyOS?